### PR TITLE
Recommend against editing the change log file

### PR DESCRIPTION
**Pull request by greg0ire**
This should prevent many conflicts between PRs.

### CODE REVIEW

File: project/CONTRIBUTING.md [hunk 1]
```diff
@@ -84,14 +84,17 @@ on your PR explaining why you feel it is the case.
 
 #### The Change log
 
-On each PR, the `CHANGELOG.md` file **has to be updated**.
+For each PR, a change log SHOULD be provided.
 
-There are few cases where the `CHANGELOG.md` file should not be touched:
+There are few cases where no change log is necessary
```
Comment (Soullivaneuh) [inline project/CONTRIBUTING.md]: Missing `:` at the end.

File: project/CONTRIBUTING.md [hunk 2]
```diff
@@ -84,14 +84,17 @@ on your PR explaining why you feel it is the case.
 
 #### The Change log
 
-On each PR, the `CHANGELOG.md` file **has to be updated**.
+For each PR, a change log SHOULD be provided.
```
Comment (Soullivaneuh) [inline project/CONTRIBUTING.md]: I would prefer `MUST`, even if it does not for few cases.

Comment (greg0ire) [inline project/CONTRIBUTING.md]: Well then I will use "must" instead of `MUST`

Comment (greg0ire) [inline project/CONTRIBUTING.md]: Otherwise people will think we refer to RFC 2119

Comment (Soullivaneuh) [inline project/CONTRIBUTING.md]: :+1:

**Comment by Soullivaneuh** [comment]
Thank you! :+1:

**Pull request merged**